src/server/templates/target.js

```diff
@@ -69,9 +74,7 @@ export default ({
 			<div id='target'>${markup}</div>
 			<script src='/js/bundle.js'></script>
 			${page ? pageScript : ''}
-			${props && script &&
-				`<script id='props' type='application/json'> ${props}</script>
-				<script src='${script}'></script>`}
+			${scriptTags}
```

Comment:
string `The value is ${null}` is actually interpolated as string "The value is null"
So, we need to add a check condition to like done with `${page ? pageScript : ''}` in the previous line.
But I think there's a better solution as which I noted above :)

Reply:
Ohh ok I see now, thanks!